### DISCUSSION TITLE
Is there a way to pipe the output of a template into another template? Or can I create a chain of prompts within a template?

### DISCUSSION THREAD

**Discussion by dvansoye** [nhaouari/obsidian-textgenerator-plugin#252] [Q&A]
I'm interested in chained prompting. If anyone has an idea how to chain prompts, please let me know.

**Comment by ncoquelet** [comment]
Hi, I'm curious too
The docuand not already played with it, but there is a run helper 
In the documentation, they mention the run helper https://docs.text-gen.com/_notes/3-+Templates/subpages/run+helper+function
And if the context inheritance is an issue for the use case, I think we can use a macro plugin like QuickAdd as an alternative
